x-pack/plugins/cases/public/components/recent_cases/index.test.tsx

```diff
@@ -27,19 +31,28 @@ const mockData = {
   ...useGetCasesMockState,
 };
 
+const useGetCurrentUserProfileMock = useGetCurrentUserProfile as jest.Mock;
```

Comment:
@cnasikas I tried only mocking the api call but it didn't seem to work. I was trying to get the api to return `undefined` or `null` but it continued to return the current user profile as defined in the `api.mocks.ts` file.

Reply:
Hey! Yes you can do

```
import * as api from '../../containers/user_profiles/api';
jest.mock('../../containers/user_profiles/api');

const spyOnBulkGetUserProfiles = jest.spyOn(api, 'bulkGetUserProfiles');

    spyOnBulkGetUserProfiles.mockImplementation(() => {
       // code
    });
```

No need to change the way you did it 🙂